bin/test-release.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -eux
+
+version=$1
+
+coursier fetch \
```

Comment:
1. Good idea to use resolve instead of fetch! 
2. I think it's OK to copy-paste the org:artifact names, the script is readable and easy to edit with multiple cursors (or visual mode in vim in my case 😄)

This doesn't need to be automatically executed, I just find it helpful to have these scripts in my repos to sanity check a release (and get a quick overview of the cross-build).